das/coordinator.go

```diff
@@ -128,7 +129,7 @@ func (sc *samplingCoordinator) getCheckpoint(ctx context.Context) (checkpoint, e
 	return newCheckpoint(stats), nil
 }
 
-// concurrencyLimitReached indicates whether concurrencyLimit has been reached
-func (sc *samplingCoordinator) concurrencyLimitReached() bool {
-	return len(sc.state.inProgress) >= sc.concurrencyLimit
+// ConcurrencyLimitReached indicates whether ConcurrencyLimit has been reached
+func (sc *samplingCoordinator) ConcurrencyLimitReached() bool {
+	return len(sc.state.inProgress) >= sc.ConcurrencyLimit
```

Comment:
Why exported? It seems like a relic from IDE find-and-replace

Reply:
Indeed